src/common/datamodeltypes/canrawsenderdata.h

```diff
@@ -0,0 +1,8 @@
+#ifndef CANRAWSENDERDATA_H
+#define CANRAWSENDERDATA_H
+
+#include "candevicedata.h"
+
+typedef CanDeviceDataIn CanRawSenderDataOut;
```

Comment:
What do you think to keep it like that? CanDevice have two data types - In and Out. CanDeviceDataIn is Out direction for sender.